webrtc.html

```diff
@@ -9849,6 +9847,34 @@ <h2>Methods</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3>canInsertDTMF algorithm</h3>
+      <p>To <dfn>determine if DTMF can be sent</dfn> for an <code><a>RTCDTMFSender</a></code>
+      instance <var>dtmfSender</var>, the user agent MUST queue a task that runs the following steps:</p>
+      <ol>
+        <li>Let <var>sender</var> be the <code><a>RTCRtpSender</a></code> associated with
+        <var>dtmfSender</var>.</li>
+        <li>Let <var>transceiver</var> be the <code><a>RTCRtpTransceiver</a></code>
+        associated with <var>sender</var>.</li>
+        <li>Let <var>connection</var> be the <code><a>RTCPeerConnection</a></code> associated with
+        <var>transceiver</var>.</li>
+        <li>If <var>connection</var>'s <code><a>RTCPeerConnectionState</a></code> is not <code>"connected"</code>
+       return <code>false</code> and abort these steps.</li>
+        <li>If <code><var>sender</var>.track</code> is <code>null</code> return <code>false</code>
+        and abort these steps.</li>
+        <li>If <code><var>sender</var>.track.kind</code> is not <code>"audio"</code> return
+        <code>false</code> and abort these steps.</li>
+        <li>If <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> slot is not set return
```

Comment:
Do we need this check given that next thing done is to check if it is sendrecv or sendonly?